packages/block-editor/src/components/block-tools/block-toolbar-popover.js

```diff
@@ -47,15 +47,19 @@ export default function BlockToolbarPopover( {
 		isToolbarForced.current = false;
 	} );
 
+	// If the block has a parent with __experimentalCaptureToolbars enabled,
+	// the toolbar should be positioned over the topmost capturing parent.
+	const clientIdToPositionOver = capturingClientId || clientId;
+
 	const popoverProps = useBlockToolbarPopoverProps( {
 		contentElement: __unstableContentRef?.current,
-		clientId,
+		clientId: clientIdToPositionOver,
```

Comment:
> Hmm, I'm still seeing the toolbar moving along the page when scrolling:

I tried to force `shift: false`, and it does have an effect for some blocks, but for some reason the header and its contents are still sticky in this branch:


https://github.com/user-attachments/assets/43408c76-b867-481b-b8d8-7751b39786a2

On trunk, all toolbars scroll out of view with the anchored element.